worksheet_B/Mandelbrot.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include "stdafx.h"
 
+const int ZoomValue = 100;
```

Comment:
This is quite an extreme zoom -- I'd have used a value around 4, but each to their own!